packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -43,6 +43,7 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
     uint256 public burnedBondPercentage; // Percentage of the bond that is paid to the UMA store if the assertion is disputed.
 
     bytes32 public constant defaultIdentifier = "ASSERT_TRUTH";
+    int256 public constant numericalTrue = 1e18; // Numerical representation of true.
```

Comment:
Should we better use capitalized version `NUMERICAL_TRUE`

Reply:
wellll...they did raise this on the `N-02 Constant not using UPPER_CASE format` and I specifically pushed back on that for the case of `defaultIdentifier` as I did not want the public interface for this to be upper case when we have `defaultCurrency` as well as `defaultLiveness` not being upper case. there is an argument to go against the solidity styling guide for consistency within our own contract